bleachbit/Action.py

```diff
@@ -206,16 +206,16 @@ def get_file(path):
         def get_walk_all(top):
             """Delete files and directories inside a directory but not the top directory"""
             for expanded in glob.iglob(top):
-                any_match = False
-                for path in FileUtilities.children_in_directory(
-                        expanded, True):
-                    any_match = True
+                for path in FileUtilities.children_in_directory(expanded, True):
                     yield path
-                # This is a lint checker because this scenario may
-                # indicate the cleaner developer made a mistake.
-                if not any_match and os.path.isfile(expanded):
-                    logger.debug(
-                        _('search="walk.all" used with regular file path="%s"'), expanded)
+                else:
```

Comment:
This else used to refer to an `if` but not refers to a `for` . I think you mean to unindent what is now in the `else`